dbclients/tantalus.py

```diff
@@ -534,29 +540,32 @@ def get_dataset_file_instances(self, dataset_id, dataset_model, storage_name, fi
         Returns:
             file_instances (list)
         """
-        if filters is None:
+
+        if filters == None:
             filters = {}
 
-        file_instances = []
+        filters["fileinstance__storage__name"] = storage_name
```

Comment:
We dont want to do this because we want to get all file resources even those that dont have a file instance on the storage we are interested in.  if the file resource doesnt have an instance on the storage, then we catch it below in the check